releasenotes/notes/fix-yaml-scrubbing-06bb43a799494065.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    The scubbing logic for configurations now scrubs YAML lists. This fixes ``flare_stripped_keys`` not working on YAML.
+    list.
```

Comment:
```suggestion
```

Reply:
We need the list, but I removed the dot on the previous line.